src/scss/share.scss

```diff
@@ -8,11 +8,11 @@
 @mixin oShareBase($classname: o-share) {
 	.#{$classname} {
 		display: inline-block;
+	}
 
-		ul {
-			margin: 0;
-			padding: 0;
-		}
+	ul {
```

Comment:
I'm moving it inside the .classname selector so it only affects the <ul> elements inside an o-share element